crates/turborepo-lib/src/run/scope/filter.rs

```diff
@@ -918,7 +918,11 @@ mod test {
             packages,
             expected
                 .iter()
-                .map(|s| WorkspaceName::Other(s.to_string()))
+                .map(|s| if ROOT_PKG_NAME.eq(*s) {
+                    WorkspaceName::Root
+                } else {
+                    WorkspaceName::Other(s.to_string())
+                })
```

Comment:
Not a big deal, but we can just use the conversion logic.
```suggestion
                WorkspaceName::from(*s)
```